applications/Chat/evaluate/eval.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import json
+
+from evaluator import Evaluator
+from utils import jload
+
+def main(args):
+     # load config
+    config=jload(args.config_file)
+
+    if config["language"] == "cn":
```

Comment:
As we only support `cn` now. You should raise proper information in `else`

Reply:
okay, I will add `else` clause.